### ISSUE TITLE
Kiosk: Upgrade to Python 3.13 version

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2998]

**Comment by urapadmin** [comment]
not possible to go up to the current 3.13 as of 08.XII.24: No rawpy wheels available.

**Comment by urapadmin** [comment]
they are available now

**Comment by urapadmin** [comment]
There requirements can't be installed anymore (not sure if they are still needed at all):
- pathtools
- install

**Comment by urapadmin** [comment]
other libraries that make trouble:
 - lxml updated to latest 6.0.1
 - greenlet updated to latest
  - cffi removed
  - pyodbc updated from 4.0.39 to lastest 5.2.0.
  - pywinpty removed
  - unmaintained: typed-ast removed

**Comment by urapadmin** [comment]
### failure: Python 3.13 not happening, yet
- we have to wait for flask_admin 2.0. At least I hope that is when flask_admin is supporting Python 3.13. Currently I get intro trouble in the contrib package that tries to import the deprecated and meanwhile dropped pkg_resources which is trying to import an ImpImporter from pkgutil - pkgutil does not have that impImporter any longer. Smells quite like they are not ready.

**Comment by urapadmin** [comment]
so, let's try Python 3.11 at least ...

**Comment by urapadmin** [comment]
Seems to work. At least my development environment is back working. We'll try that on Beset,

**Comment by luizaogs** [comment]
What I have tested with Python 3.11 installed: 

- hub: reset dock and prepare
- file repository: filters
- file repository: changed context of photo
- query and view: views
- query and view: queries
- director's view: new modification
- director's view: old stuff also shows up in 2020
- stratigraphy studio: different modes

**Comment by luizaogs** [comment]
I'm trying to do a synchronization cycle but the hotel wifi is not cooperating.

If you want any other corners poked, let me know.

**Comment by urapadmin** [comment]
you are in hotels too much 🐑  synchronization cycle would be nice, indeed, but it isn't urgent.

**Comment by luizaogs** [comment]
I’ll have better wifi tomorrow 🐄

**Comment by luizaogs** [comment]
Synchronization did not work.

<img width="1498" height="231" alt="Image" src="https://github.com/user-attachments/assets/063aa8ef-7255-4526-a721-437f85a746b4" />

**Comment by urapadmin** [comment]
I don*'t believe it. There might have been some piece of FM stuck from a former task. Please try again.

**Comment by luizaogs** [comment]
You were right, it's fine.

**Comment by urapadmin** [comment]
good. Thanks.

**Comment by luizaogs** [comment]
Do you want anything else?

**Comment by urapadmin** [comment]
no, that should do. We leave this open for the day when I actually can upgrade to Python 3.13. That day was not there, yet.

- Will be applied to all of Beset soon: #3303
